recipes/msmpi-bin/bld.bat

```diff
@@ -0,0 +1,3 @@
+start "" /WAIT MSMPISetup.exe -installroot %LIBRARY_PREFIX%
```

Comment:
Need `-unattend` here to avoid opening dialog and waiting for input.  After I added this, I was able to get build to complete, but tests were failing because files put in %LIBRARY_BIN% by the installer did not end up in the test env, while the file below put there in COPY did. I don't know why that would be.